database-mongodb/MySQL/Images.js

```diff
@@ -0,0 +1,66 @@
+const connection = require('./connect.js');
+const { promisifyAll } = require('bluebird');
+
+promisifyAll(connection);
+
+connection.connect((err) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log('Successfully connected to MySQL DB');
+  }
+});
+
+
+function insertRecords (records) {
+  let insertionsParsedToMySQLSyntax = '';
+  for (let i = 0; i < records.length; i++) {
+    if (i !== 999) {
+      insertionsParsedToMySQLSyntax = insertionsParsedToMySQLSyntax + "('" + records[i].itemId+ "','" + records[i].itemImages+ "'),";
+    } else {
+      insertionsParsedToMySQLSyntax = insertionsParsedToMySQLSyntax + "('" + records[i].itemId+ "','" + records[i].itemImages+ "')";
+    }
+
+  }
+  const query = `INSERT INTO images (itemId,itemImages) VALUES${insertionsParsedToMySQLSyntax}`;
+
+  return connection.queryAsync(query);
```

Comment:
So this returns a promise? Is error handling necessary here?

Reply:
My understanding is that those promise objects are being placed in an array that, itself, will be passed into Promise.all which then will catch the error on the first individual promise that fails. Is that your understanding as well?